compiler/src/dmd/cli.d

```diff
@@ -336,6 +336,15 @@ dmd -cov -unittest myprog.d
             With $(I filename), write module dependencies as text to $(I filename)
             (only imports).`,
         ),
+        Option("dllimport=<value>",
+            "Windows only: select symbols to dllimport (none/defaultLibsOnly/all)",
+            `Which global variables to dllimport implicitly if not defined in a root module
+            $(UL
+                $(LI $(I none): None
+                $(LI $(I defaultLibsOnly): Only druntime/Phobos symbols
+                $(LI $(I all): All
```

Comment:
I think you are missing some closing `)` here

Reply:
Indeed, fixed now.